.github/workflows/build_python_3.yml

```diff
@@ -1,9 +1,6 @@
 name: Build Python 3
 
 on:
-  merge_group:
```

Comment:
I think this file isn't actually run? I added it originally because I thought this was THE build job, but turns out that's the `build_deploy` job above. So I'm just cleaning this up

Reply:
As for `rust-ci.yml`, I just missed it. Thanks for catching 👍

Reply:
Actually, I think that triggers on any push to any branch. I see it running on all branches already